### DISCUSSION TITLE
Welcome to Radix Dev(Open Source) Discussions!

### DISCUSSION THREAD

**Discussion by mozhi-bateman** [mozhi-bateman/radix-dev-quick-starter#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oscarscfrederiksen** [comment]
Hey! Thanks for this awesome space! I just came here to hopefully get some contributions to my repo and contribute to others. I recently started compiling a repo called "Useful-Packages" which aims to be a massive composition of everyone's favourite packages, APIs and the like for all languages, systems and platforms. It would be awesome if you could check it out and contribute; the link is: https://github.com/oscarscfrederiksen/Useful-Packages.
